.travis.yml

```diff
@@ -1,16 +1,14 @@
 language: python
-python:
-  - 2.6
-  - 2.7
-  - 3.2
-  - 3.3
-  - 3.4
-  - pypy
-  - pypy3
 env:
-  - PYTEST=2.6.4
-  - PYTEST=2.7.2
+  - TOXENV=py26
+  - TOXENV=py27
+  - TOXENV=py32
+  - TOXENV=py33
+  - TOXENV=py34
```

Comment:
There should, but the way this works it depends on the default Python image in Travis having all of these versions available. At the moment, Python 3.5 is not available. See https://github.com/travis-ci/travis-ci/issues/4794. There is a workaround it seems, or I could make this expected to fail for now.